content/en/stateless-apps.md

```diff
@@ -7,30 +7,29 @@ tags: ["fundamental", "application", ""]
 
 ## What it is
 
-A stateless application doesn’t save any client session (state) data on the server where the application lives. 
-Each session is carried out as if it was the first time and responses are not dependent upon data from a previous session and 
-provides functionality to use print services, CDN (Content Delivery Network) or the Web Servers 
-in order to process every short-term request. 
-For example, someone is searching a question in the search engine and pressed the Enter button. 
-In case if the searching operation gets interrupted or closed due to some reason, 
-you have to start a new one as there is no saved data for your previous request.
+Stateless applications process requests as if they were their first. 
+The app doesn't "remember" previous interactions or user session data. 
+Data from previous interactions is referred to as state, and since that data isn't stored anywhere, these apps are state*less*. 
+In short, stateless apps lack permanent storage. 
+Here's an example. 
+When you use a search engine, and that search is interrupted (e.g., the window is closed), those search results are lost. 
+You'll need to start all over.
+
+On the other hand, applications that process requests while considering previous interactions are called [stateful applications](/stateful-apps/). 
 
 ## Problem it addresses
 
-Stateless applications tackle the problem of resiliency, 
-because different pods across a [cluster](/cluster/) can work independently, 
-with multiple requests coming to them at the same time. 
-If there’s a problem, you can easily restart the application, 
-and it will return to its initial state with little or no downtime. 
-As such, the benefits of stateless applications include resiliency, elasticity, and high availability. 
-However, most applications we use today are at least partly [stateful](/stateful-apps/), 
-as they store things like preferences and settings to improve the user experience.
+When stateful applications, apps that *do* have session data, are horizontally scaled, every instance needs access to that same data.
+If one instance processes a request, the other instances must have access to that session data to take it into account when they process the next request. 
+And that brings some challenges. 
+All instances would have to somehow sync that session data, which often used to be stored only in-memory of an instance before the rise of the cloud.  
```

Comment:
```suggestion
For example, all instances must sync that session data stored *in memory* of the instance that processed the request.
```
I committed your changes, @iamNoah1, but I'm confused by this statement. This sounds like the stateful way is the old way of doing things in a pre-cloud world. Lots of apps are stateful, so I changed the language to be more affirmative ("this is how it is with stateful"), but it's also the case on the cloud, isn't it? We aren't comparing pre-cloud vs cloud, but stateless vs stateless. 

Also edited the other sentences in this section but don't think I changed the meaning.